notebooks/406-3D-pose-estimation-webcam/406-3D-pose-estimation.ipynb

```diff
@@ -0,0 +1,592 @@
+    "# load the model on the CPU (you can use GPU or MYRIAD as well)\n",
+    "compiled_model = ie_core.compile_model(model=model, device_name=\"CPU\")\n",
+    "infer_request = compiled_model.create_infer_request()\n",
+    "input_tensor_name = model.inputs[0].get_any_name()\n",
+    "\n",
+    "# get input and output names of nodes\n",
+    "input_layer = compiled_model.input(0)\n",
+    "output_layers = list(compiled_model.outputs)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5c0ffd17-df71-4178-8df8-db4ccf431621",
+   "metadata": {},
+   "source": [
+    "Input of the model is the data of the input image and the outputs are heatmaps, PAF (part affinity fields) and features"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e1b25847-fc80-41a1-930b-7c304fd1fe70",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "input_layer.any_name, [o.any_name for o in output_layers]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "48eb5032-a06e-48c1-a3d6-f0fbad9924fb",
+   "metadata": {},
+   "source": [
+    "## Processing\n",
+    "### Model Inference\n",
+    "Input image or frames captured from video files or live webcam is used as the input of the 3D model to obtain the output heatmaps, PAF (part affinity fields) and features."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "08f8055b-a6cf-4003-8232-6f73a86d6034",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def model_infer(scaled_img, stride):\n",
+    "    \"\"\"\n",
+    "    Run model inference on the input image\n",
+    "\n",
+    "    Parameters:\n",
+    "        scaled_img: resized image according to the input size of the model\n",
+    "        stride: int, the stride of the window\n",
+    "    \"\"\"\n",
+    "\n",
+    "    # Remove excess space from the picture\n",
+    "    img = scaled_img[\n",
+    "        0 : scaled_img.shape[0] - (scaled_img.shape[0] % stride),\n",
+    "        0 : scaled_img.shape[1] - (scaled_img.shape[1] % stride),\n",
+    "    ]\n",
+    "\n",
+    "    img = np.transpose(img, (2, 0, 1))[\n",
+    "        None,\n",
+    "    ]\n",
+    "    infer_request.infer({input_tensor_name: img})\n",
+    "    # A set of three inference results is obtained\n",
+    "    results = {\n",
+    "        name: infer_request.get_tensor(name).data[:]\n",
+    "        for name in {\"features\", \"heatmaps\", \"pafs\"}\n",
+    "    }\n",
+    "    # Get the results\n",
+    "    results = (results[\"features\"][0], results[\"heatmaps\"][0], results[\"pafs\"][0])\n",
+    "\n",
+    "    return results"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6991403a-4f87-45be-9b3f-d30b23a46dbe",
+   "metadata": {},
+   "source": [
+    "### Draw 2D Pose Overlays\n",
+    "We need to define some connections between the joints in advance, so that we can draw the structure of the human body in the result image when obtaining the inference results.\n",
```

Comment:
```suggestion
    "We need to define some connections between the joints in advance, so that we can draw the structure of the human body in the resulting image after obtaining the inference results.\n",
```